digitalpy/core/parsing/abstract_format.py

```diff
@@ -1,13 +1,13 @@
 from abc import abstractmethod
 import importlib
 from pickle import PERSID
-from digitalpy.parsing.format import Format
-from digitalpy.routing.request import Request
-from digitalpy.routing.response import Response
-from digitalpy.core.object_factory import ObjectFactory
-from digitalpy.model.object_id import ObjectId
-from digitalpy.model.node import Node
-from digitalpy.model.build_depth import BuildDepth
+from digitalpy.core.parsing.format import Format
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0